release-notes/release-notes571.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: default
+title: TinyMCE 5.7.1
+title_nav: TinyMCE 5.7.1
+description: Release notes for TinyMCE 5.7.1
+keywords: releasenotes bugfixes
+---
+
+## Overview
+
+{{site.productname}} 5.7.1 was released for {{site.enterpriseversion}} and {{site.cloudname}} on Wednesday, March 24<sup>th</sup>, 2021. It includes {{site.productname}} 5.7.1 and additional changes to premium plugins. These release notes provide an overview of the changes for {{site.productname}} 5.7.1, including:
+
+- [Accompanying Premium Plugin changes](#accompanyingpremiumpluginchanges)
+- [General bug fixes](#generalbugfixes)
+- [Security fixes](#securityfixes)
+- [Upgrading to the latest version of TinyMCE 5](#upgradingtothelatestversionoftinymce5)
+
+{{site.releasenotes_for_stable}}
+
+## Accompanying Premium Plugin changes
+
+The following premium plugin updates were released alongside {{site.productname}} 5.7.1.
+
+### PowerPaste 5.4.2
+
+The {{site.productname}} 5.7.1 release includes an accompanying release of the **PowerPaste** premium plugin.
+
+**PowerPaste** 5.4.2 provides the following bug fixes:
+
+* Fixed a security issue where dragging and dropping content wasn't correctly sanitized in some cases.
+
+For information on the PowerPaste plugin, see: [PowerPaste plugin]({{site.baseurl}}/plugins/premium/powerpaste/).
+
+## General bug fixes
+
+{{site.productname}} 5.7.1 provides fixes for the following bugs:
+
+* Fixed the `help` dialog incorrectly linking to the changelog of TinyMCE 4 instead of TinyMCE 5.
+* Fixed a bug where error messages were displayed incorrectly in the image dialog.
+* Fixed an issue where URLs were not correctly filtered in some cases.
```

Comment:
This is a security issue so it should be in the section below.

Reply:
Fixed in [`d2acacf` (#1907)](https://github.com/tinymce/tinymce-docs/pull/1907/commits/d2acacf2843abcba4d696f88d45c4672f37f1ce6)